versioned_docs/version-0.2.0/contribute-to-kubeslice-docs.mdx

```diff
@@ -47,6 +47,6 @@ No contribution is too big or small! However, to ensure that the community deriv
 
 - Focus on providing a clear description of the issue. Some key points to consider would be specifically describing what is missing, outdated, erroneous, or requires qualitative/technical improvement.
 - Detail the specific impact the issue has on users.
-- Delimit the scope of the issue. If the scope is larger, we'd request you to break it down into smaller tasks within an issue. For example, **Creating a Contribution Guide** is very wide in scope since there would be multiple tasks associated with the issue. However,**Fixing a grammatical error on the Quickstart page** is a more narrowly scoped issue that would, potentially, require only a single pull request.
+- Delimit the scope of the issue. If the scope is larger, we'd request you to break it down into smaller tasks within an issue. For example, **Creating a Contribution Guide** is very wide in scope since there would be multiple tasks associated with the issue. However,**Fixing a grammatical error on the quick start page** is a more narrowly scoped issue that would, potentially, require only a single pull request.
```

Comment:
```suggestion
- Delimit the scope of the issue. If the scope is larger, we'd request you to break it down into smaller tasks within an issue. For example, **Creating a Contribution Guide** is very wide in scope since there would be multiple tasks associated with the issue. However,**Fixing a grammatical error on the Quickstart page** is a more narrowly scoped issue that would, potentially, require only a single pull request.
```